LEGO1/mxdsaction.cpp

```diff
@@ -3,13 +3,18 @@
 #include <float.h>
 #include <limits.h>
 
+#define TWOCC(a, b) (((a) << 0) | ((b) << 8))
```

Comment:
Minor code maintenance idea: might be worth combining this and `FOURCC` from `mxdsfile.cpp` into one global header, perhaps `mxtypes.h`